app/web/src/store/components.store.ts

```diff
@@ -1168,6 +1286,19 @@ export const useComponentsStore = (forceChangeSetId?: ChangeSetId) => {
             });
           },
 
+          async UPDATE_COMPONENT_ATTRIBUTES(
+            componentId: ComponentId,
+            payload: UpdateComponentAttributesArgs,
+          ) {
+            return new ApiRequest({
+              method: "put",
+              url: `v2/workspaces/${workspaceId}/change-sets/${changeSetId}components/${componentId}/attributes`,
```

Comment:
This is missing  a /:
```suggestion
              url:
                `v2/workspaces/${workspaceId}/change-sets/${changeSetId}/components/${componentId}/attributes`,
```